bors.toml

```diff
@@ -1 +1 @@
-status = ["continuous-integration/travis-ci/push"]
+status = ["build"]
```

Comment:
I think this must be `build / Build and Test (push)`.

I don't know where to find this easily. I clicked on the "code" tab of the repo, changed the branch to trying. There is a green check sign next to the commit. If I click that, I see "build / Build ..." abbrevated. I used firefox' "Inspect element" to see the full text...

Reply:
did you try this?